services/src/test/java/org/exoplatform/onlyoffice/OnlyofficeEditorServiceTest.java

```diff
@@ -44,175 +43,276 @@
+     * Test create editor for incorrect file
+     */
+    @Test
+    public void testCreateEditorIncorrectFile() throws Exception {
+        startSessionAs("john");
+        Node node = createDocument("Test Document.docx", "nt:base", "testContent");
+        try {
+            editorService.createEditor("http", "127.0.0.1", 8080, "john", null, node.getUUID());
+        } catch (OnlyofficeEditorException e) {
+            // Ok
+            node.remove();
+            session.save();
+            return;
+        }
+        // Fail if the exception wasn't thrown
+        fail();
+    }
 
-    DocumentContent documentContent = editorService.getContent("john", config.getDocument().getKey());
+     /**
+     * Test force save
+     * No test for output
+     */
+    @Test
+    public void testForceSave() throws Exception {
+        startSessionAs("john");
+        editorService.forceSave("john", "key", false, false, true, "testForceSave");
+    }
 
-    assertNotNull(documentContent);
-    String content = IOUtils.toString(documentContent.getData(), "UTF-8");
-    assertEquals("testContent", content);
-    assertEquals("application/vnd.oasis.opendocument.text", documentContent.getType());
-    node.remove();
-  }
+    /**
+     * Test get content
+     */
+    @Test
+    public void testGetContent() throws Exception {
+        startSessionAs("john");
+        Node node = createDocument("Test Document.docx", "nt:file", "testContent");
+        Config config = editorService.createEditor("http", "127.0.0.1", 8080, "john", null, node.getUUID());
+
+        DocumentContent documentContent = editorService.getContent("john", config.getDocument().getKey());
+
+        assertNotNull(documentContent);
+        String content = IOUtils.toString(documentContent.getData(), "UTF-8");
+        assertEquals("testContent", content);
+        assertEquals("application/vnd.oasis.opendocument.text", documentContent.getType());
+        node.remove();
+    }
 
-  @Test
-  public void testIsDocumentMimeSupported() throws Exception {
-    startSessionAs("john");
-    Node node = createDocument("Test Document.docx", "nt:file", "testContent");
-    assertTrue(editorService.isDocumentMimeSupported(node));
-    node.getNode("jcr:content").setProperty("jcr:mimeType", "text/plain");
-    session.save();
-    assertFalse(editorService.isDocumentMimeSupported(node));
-    Node unsctructuredNode = session.getRootNode().addNode("Test Document.docx", "nt:unstructured");
-    session.save();
-    assertTrue(editorService.isDocumentMimeSupported(unsctructuredNode));
-    node.remove();
-    unsctructuredNode.remove();
-  }
+    /**
+     * Test get document with workspace and path
+     */
+    @Test
+    public void testGetDocument() throws Exception {
+        startSessionAs("john");
+        Node node = createDocument("Test Document.docx", "nt:file", "testContent");
+        Node nodeDocument = editorService.getDocument(null, node.getPath());
+        assertNotNull(nodeDocument);
```

Comment:
We should check more deeply if the document is correct (assert document title, node type, ...).